File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_GroupCommandsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_GroupCommandsService : BaseService<Sys_GroupCommands, ISys_GroupCommandsRepository>
    , ISys_GroupCommandsService
    {
    private readonly ISys_GroupCommandsRepository _Sys_GroupCommandsRepository;
    public Sys_GroupCommandsService(ISys_GroupCommandsRepository Sys_GroupCommandsRepository) : base(Sys_GroupCommandsRepository)
    {
    _Sys_GroupCommandsRepository = Sys_GroupCommandsRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_GroupCommands.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_GroupCommands } from './Sys_GroupCommands.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_GroupCommandsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Group_Id', 'Command_Id', 'Function_Id', 'Menu_Id'];

    List: Sys_GroupCommands[] | undefined;
    ListFilter: Sys_GroupCommands[] | undefined;
    FormData!: Sys_GroupCommands;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_GroupCommands";
    }
}

